packages/x-components/src/x-modules/history-queries/components/my-history.vue

```diff
@@ -1,58 +1,78 @@
-      <slot name="suggestion" v-bind="{ suggestion, index }">
-        <HistoryQuery
-          :suggestion="suggestion"
-          data-test="history-query-item"
-          class="x-history-queries__item"
-        >
-          <template #default>
-            <!-- eslint-disable max-len -->
-            <!--
+          <slot name="suggestion" v-bind="{ suggestion, index }">
+            <HistoryQuery
+              :suggestion="suggestion"
+              data-test="history-query-item"
+              class="x-history-queries__item"
+            >
+              <template #default>
+                <!-- eslint-disable max-len -->
+                <!--
               @slot History Query content
                   @binding {Suggestion} suggestion - History Query suggestion data
                   @binding {number} index - History Query suggestion index
             -->
-            <!-- eslint-enable max-len -->
-            <slot name="suggestion-content" v-bind="{ suggestion, index }" />
-          </template>
-          <template #remove-button-content>
-            <!--
+                <!-- eslint-enable max-len -->
+                <slot name="suggestion-content" v-bind="{ suggestion, index }">
+                  <HistoryTiny class="x-icon--l" />
+                  <div class="x-list x-list--vertical">
+                    <span data-test="my-history-query">{{ suggestion.query }}</span>
+                    <span data-test="my-history-time">{{ formatTime(suggestion.timestamp) }}</span>
+                  </div>
+                </slot>
+              </template>
+              <template #remove-button-content>
+                <!--
               @slot History Query remove button content
                   @binding {Suggestion} suggestion - History Query suggestion data
                   @binding {number} index - History Query suggestion index
             -->
-            <slot name="suggestion-remove-content" v-bind="{ suggestion, index }" />
-          </template>
-        </HistoryQuery>
-      </slot>
-    </template>
-  </BaseSuggestions>
+                <slot name="suggestion-remove-content" v-bind="{ suggestion, index }" />
+              </template>
+            </HistoryQuery>
+          </slot>
+        </template>
+      </BaseSuggestions>
+    </li>
+  </ul>
 </template>
 
 <script lang="ts">
   import { HistoryQuery } from '@empathyco/x-types';
   import Vue from 'vue';
-  import { Component, Prop } from 'vue-property-decorator';
+  import { Component, Inject, Prop } from 'vue-property-decorator';
+  import HistoryTiny from '../../../components/icons/history-tiny.vue';
   import { xComponentMixin } from '../../../components/x-component.mixin';
   import { State } from '../../../components/decorators/store.decorators';
   import BaseSuggestions from '../../../components/suggestions/base-suggestions.vue';
+  import { groupItemsBy } from '../../../utils/index';
+  import { SnippetConfig } from '../../../x-installer/index';
```

Comment:
Avoid importing from barrel

```suggestion
  import { SnippetConfig } from '../../../x-installer/api/api.types';
```